charts/core/src/Chart/Chart.styles.ts

```diff
@@ -0,0 +1,22 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import {
+  borderRadius,
+  color,
+  InteractionState,
+  Variant,
+} from '@leafygreen-ui/tokens';
+
+export const chartStyles = css`
+  grid-row: 2;
```

Comment:
Completely agree! Right now this isn't really doing much since the header doesn't exist yet. But I'll switch this to `grid-area` when we add in the header.